rc/tools/spell.kak

```diff
@@ -3,6 +3,8 @@ declare-option -hidden str spell_last_lang
 declare-option -hidden str spell_tmp_file
 
 declare-option -docstring "default language to use when none is passed to the spell-check command" str spell_lang
+declare-option -docstring "spell checking engine to use" str spell_cmd 'aspell --byte-offsets -a -l'
```

Comment:
I think this default command would force users to pass a language to `:spell`, because it is trailed with `-l`.

Reply:
I will make a v2 of this patch with a couple of if statements for setting up the backend. Maybe we could try the backends and use the first one found, something similar we do with windowing.

Thanks for the awesome feedback!